src/client/components/page-layout/page-layout.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div class="page-layout">
+    <page-header :title="page.title" :subtitle="page.subtitle" :image="page.coverImage" />
+
+    <div class="page-layout__body">
+      <div class="page-layout__sidebar" v-if="page.hasToc && page.tocItems.length">
+        <table-of-contents :items="page.tocItems" />
+      </div>
+
+      <div class="page-layout__sections">
+        <content-section v-for="(section, index) in page.sections" :key="index" :section="section" :id="section.slug" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { ContentSection, TableOfContents, PageHeader } from '../'
+
+export default {
+  components: { ContentSection, TableOfContents, PageHeader },
+  
+  props: {
+    page: { type: Object, required: true }
+  },
+}
+</script>
+
+<style>
+@import '../app-core/index.css';
+
+.page-layoug__sections{
```

Comment:
fix typo, `layout` not `layoug`. also make sure there's a space between `sections` and `{`